.themes/default/source/arcs.html

```diff
@@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
       {% include rp.html rp=arc_rp %}
     {% endfor %}
     </ul>
-  {% else %}
-    {% include rp.html rp=rp %}
```

Comment:
Can you make an issue for the Arc View sorting being confusing / needing to
be improved?

On Wed, Jul 8, 2015, 15:25 derpynirvash <notifications@github.com> wrote:

> In .themes/default/source/arcs.html
> <https://github.com/xiagu/jekyll-rp_logs/pull/19#discussion_r34188202>:
>
> > @@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
> >        {% include rp.html rp=arc_rp %}
> >      {% endfor %}
> >      </ul>
> > -  {% else %}
> > -    {% include rp.html rp=rp %}
>
> Eh, in my opinion, it at least needs to be sorted better.
>
> But sounds fine, leaving design decisions for later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xiagu/jekyll-rp_logs/pull/19/files#r34188202>.
>